src/main/resources/l10n/JabRef_en.properties

```
@@ -2579,4 +2579,4 @@ Failed\ to\ download\ from\ URL=Failed to download from URL
 Finished=Finished
 Finished\ writing\ metadata\ for\ library\ %0\ (%1\ succeeded,\ %2\ skipped,\ %3\ errors).=Finished writing metadata for library %0 (%1 succeeded, %2 skipped, %3 errors).
 Processing...=Processing...
-Writing\ metadata\ to\ {}=Writing metadata to {}
+Writing\ metadata\ to\ %0=Writing metadata to {}
```

```suggestion
Writing\ metadata\ to\ %0=Writing metadata to %0
```
replacement on both sides